docs/UserGuide.md

```diff
@@ -170,47 +189,75 @@ Your list of medications has been successfully shown!
 ### Daily medications:
 Displays an overview of the list of medications that you will be taking for the day.
 
-Format: `list -t today`
-* If you want to see for only a particular period of the day, simply enter `list -t today` followed by:
+Example: `meditracker> list -t today`
+* If you want to specify a particular period of the day, simply enter `list -t today` followed by:
   * `-m`: Morning (Midnight to 12 Noon)
   * `-a`: Afternoon (12pm to 6pm)
   * `-e`: Evening (6pm to Midnight)
-  * Eg: `list -t today -m` - will display the list of medications to be taken in the morning only
+  * Example: `meditracker> list -t today -m` - will display the list of medications to be taken in the morning only
 
 Output:
 ```
-____________________________________________________________
 Here are the Daily Medications you have to take today: 
 Morning:
 	1. [ ] Medication_A | 2.0
 Afternoon:
 	1. [ ] Medication_A | 1.0
-____________________________________________________________
 ```
 
+<div class="info-box">
+:information_source: <strong>Info: </strong>
+
+If you want to mark the medication you have taken, you can refer to the <a href="#take-a-medication"><code>Take 
+Command</code></a> here.
+
+</div>
 
 ## Viewing medications: `view`
 You can view detailed information about the medications you have added in the medication list.
 
-> Recommendation: Perform [`list -t all`](#All-medications) first to display the lists of medications 
-> in medication list and use its fields for the features below.
+<div class="info-box">
+:information_source: <strong>Info: </strong>
+
+Perform <a href="#all-medications"><code>list -t all</code></a> first to display the lists of medications
+in medication list and use its fields for the features below.
+
+</div>
 
 This output will be shown if you used more than one flag and argument.
 ```
 ERROR: You can only have one flag!
 ```
-> Tip: You are only allowed to use one flag and argument.
-> 
+<div class="tip-box">
+:bulb: <strong>Tip: </strong>
+You are only allowed to use one flag and argument.
+</div>
+
+```
+Usage:
+	view [-l listIndex] [-n name] [-q quantity] [-e expirationDate] [-r remarks] [-h]
```

Comment:
This part is too long, it has caused a scrollbar to appear on the UG. Consider introducing a "newline" or extracting this part out?

Reply:
You're right. The words get cut off when converted to PDF. I have brought the words to the next line in  aff146c, and also modified for the code block in "modify" section.